luarules/gadgets/game_autocolors.lua

```diff
@@ -232,59 +236,59 @@ if gadgetHandler:IsSyncedCode() then
 
     if #teamList == #allyTeamList and teamCount > 2 then
         isFFA = true
-    elseif not TeamColors[allyTeamCount] then
+    elseif not teamColors[allyTeamCount] then
         isFFA = true
     end
 
     local ffaColorNum = 1 -- Starting from color #1
     local ffaColorVariation = 0 -- Current color variation
-    local ColorVariationDelta = 128 -- Delta for color variation
+    local colorVariationDelta = 128 -- Delta for color variation
     local allyTeamNum = 0
-    local TeamSizes = {}
+    local teamSizes = {}
 
-    local function Hex2RGB(hex)
+    local function hex2RGB(hex)
         hex = hex:gsub("#","")
         return {tonumber("0x"..hex:sub(1,2)), tonumber("0x"..hex:sub(3,4)), tonumber("0x"..hex:sub(5,6))}
     end
```

Comment:
This duplicates the definition on lines 12-15

Reply:
Wait I was sure I moved it, not copied :( Thanks!